app/__init__.py

```diff
@@ -30,5 +30,9 @@ def create_app(test_config=None):
     migrate.init_app(app, db)
 
     # Register Blueprints here
+    from app.routes.routes import tasks_bp
+    app.register_blueprint(tasks_bp)
+    from app.routes.goal_routes import goals_bp
```

Comment:
Nice choice to split up the routes into files that are more specific to the resources they work with!